app/assets/stylesheets/_variables.scss

```diff
@@ -35,6 +35,6 @@ $picture-small: 75px !default;
 $picture-medium: 100px !default;
 
 //## Variables for icons or logos used in Coursemology.
-$course-achievement-badge-thumb:   $picture-thumb !default;
-$course-achievement-badge-small:   $picture-small !default;
-$course-achievement-badge-medium:  $picture-medium !default;
+$course-achievement-badge-form:   $picture-small !default;
+
+$course-logo-form: $picture-small !default;
```

Comment:
Do we have to worry about whether the variable name is too long? If not, course-admin-form-logo seems best, or even `course-admin-form-logo-height` and  `course-admin-form-logo-width` If we want to be specific.